### ISSUE TITLE
Change export style of HTML

### ISSUE THREAD

**Issue by berndhahnebach** [wereturtle/ghostwriter#527]
This might not be the right place to ask, but I could not found any better. There is neither a forum nor a mailing list on the project webpage. Thus I ask here.

I would like to have a exported html which looks similar to how github renders markdown or how it is rendered in live preview of ghostwriter. For me the HTML export of ghostwriter looks totally different than live preview opf ghostwriter. 

Could this be archived with ghostwriter?

cheers bernd

**Comment by berndhahnebach** [comment]
Noone here who knows how to change the html style or who know where I could ask for help in this regard?

cheers bernd

**Comment by MakerStuff** [comment]
Same error here.

For now i just downloaded (the style sheet from the repo)[https://github.com/wereturtle/ghostwriter/blob/master/resources/github.css], put it in the folder i exported my doc to and manually replaced `<style type="text/css">code{white-space: pre;}</style>` (for me its in line 8) with `<link rel="stylesheet" type="text/css" href="github.css">`

That is no long term solution especially as I'd like to export my formatted file as pdf, which would require more manual html and another export via web browser, but it is at least something.

I don't have enough experience in coding C++ so i cannot make any changes to the code by myself.

**Comment by wereturtle** [comment]
Thank you for your interest!  Unfortunately, I am not accepting new feature requests at this time. 
 Please read the [Contributing guide](https://github.com/wereturtle/ghostwriter/blob/master/CONTRIBUTING.md) for further details.

**Comment by berndhahnebach** [comment]
Thanks for the workaround. That helps a lot.

I found another workaround myself. 

The tool is called grip (Grip -- GitHub Readme Instant Preview) and can be found on https://github.com/vyk2rr/grip. Since I have anaconda installed it was easy to install it too. Works quite well. Run on a md file, and it will be displayed in the browser. If you edit the md file you just need to reload the browser page on the port it is running. For pdf export I use the print tool of the browser.

The contrary of this method ... I switched from ghostwriter to N++ for editing md files. 

cheers bernd
